shared/menubar/dumb.desktop.js

```diff
@@ -0,0 +1,60 @@
+// @flow
+
+import Menubar from './index.render'
+import type {RenderProps, FolderInfo} from './index.render'
+import type {DumbComponentMap} from '../constants/types/more'
+
+const folder1: FolderInfo = {
+  type: 'folder',
+  folderName: 'max,chris',
+  isPublic: true,
+  isEmpty: false,
+  openFolder: () => {}
+}
+
+const folderLong: FolderInfo = {
+  type: 'folder',
+  folderName: 'max,chris,marcopolo,patrick,strib,mgood,zanderz,gabrielh,chrisnojima,cbostrander,alness,akalin',
+  isPublic: true,
+  isEmpty: false,
+  openFolder: () => {}
+}
+
+const folder2: FolderInfo = {
+  type: 'folder',
+  folderName: 'mgood,strib,marcopolo',
+  isPublic: true,
+  isEmpty: false,
+  openFolder: () => {}
+}
+
+const propsNormal: RenderProps = {
```

Comment:
Yes it appears to be necessary. For instanc,e changing `username` to `true` with the type annotation removed does not result in an error, whereas it will with the annotation in place.